tokens.json

```diff
@@ -236,5 +236,12 @@
     "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/zks-lg.png",
     "symbol": "ZKS",
     "account": "zkstokensr4u"
-  }
+  },
+    {
+    "name": "Crypto Peso",
+    "logo": "https://eosvenezuela.io/psologo130.png",
```

Comment:
please try to attach these files to the repository (same than other tokens), so we dont depend on third party domains. It makes things a bit easier for websites (CORS and so on..) 👍

Reply:
Thank you @kesar .. created a new PR with the logos attached.